hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -299,6 +299,14 @@
       default: "${cfg.bytes_per_bank}",
       local: "true"
     },
+
+    // hex value of 0xa26a38f7
+    { name:      "ExecEn",
+      desc:      "Constant value that enables flash execution",
+      type:      "int"
+      default:   "${int("0xa26a38f7", 16)}",
```

Comment:
So, if `default` field supports `0x....` value type, then we don't need to code like this. I think it will be a small fix in the topgen. Let me check.

Reply:
ah yeah, but you'd also need to update code in `c.py` so that they generate the headers.. i just got lazy :). 
thanks for having a look though!